jenkins-build-pack-api/Jenkinsfile

```diff
@@ -707,6 +785,10 @@ def loadBuildModules(buildModuleUrl){
 		configLoader = configModule.initialize(resultJsonString)
 		echo "config loader loaded successfully."
 
+		awsModule = load "aws-apigateway.groovy"
```

Comment:
Can we rename this to awsAPIGatewayModule? We will have awsLambdaModule soon. awsModule is too generic.